File: aisile-parent/aisile-manager-web/src/main/webapp/js/service/brandService.js
    app.service('brandService',function($http){
    	//查询全部
    	this.findAll=function(){
    		return  $http.get('../brand/findAll.do');
    	}
    	//分页
    	this.findPage=function(page,rows){
    	   return $http.get('../brand/findPage.do?page='+page+'&rows='+rows)
    	} 
    	//条件查询
    	this.findSearch=function(page,rows,searchEntity){
    		return $http.post('../brand/findSearch.do?page='+page+"&rows="+rows,searchEntity)
    	}
    	//保存
    	this.update =function(entity){
    		return $http.post('../brand/update.do',entity)
    	}
    	this.addBrand =function(entity){
    		return $http.post('../brand/addBrand.do',entity)
    	}
    	//查询实体
    	this.findOne=function(id){
    		return $http.get('../brand/findOne.do?id='+id)
    	}
    	//删除
    	this.dele=function(selectIds) {
    		return $http.get('../brand/delete.do?ids='+selectIds)
    	}
    	//下拉列表数据
    	this.selectOptionList=function(){
    		return $http.get('../brand/selectOptionList.do');
    	}

    })

File: aisile-parent/aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/BrandService.java
package com.aisile.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.aisile.pojo.TbBrand;
import com.aisile.pojo.entity.pageResult;


public interface BrandService {
	//查询全部
	public List<TbBrand> findAll();
	//分页
	public pageResult findPage(int pageNum,int pageSize);
	//增加
	public void add(TbBrand tbBrand);
	//修改
	public void update(TbBrand brand);
	//根据id找值
	public TbBrand findOne(Long id);
	//批删
	public void delete(Long [] ids);
	/*
	 * 条件查询
	 * */
	public pageResult findSearch(TbBrand brand, int pageNum,int pageSize);
	//品牌下拉框
	List<Map> selectOptionList();
}
